pkg/controller/framework/controller_test.go

```diff
@@ -32,7 +32,12 @@ import (
 	"github.com/google/gofuzz"
 )
 
-func Example() {
```

Comment:
Ah, ok.  Thanks... 
- Yes I guess these are meant to be "pure" examples.  I didn't know about this feature.
- I'm guessing if that's the case, there is no need to consolidate the output comments.
Possibly the only change we should persist  is the more explicit naming.
Closing this PR.